### PR TITLE
Simplify the WCA Deceased Registered Speedcubers Policy

### PR DESCRIPTION
**Pull request by Nanush7**
"Vital status" used as suggested [here](https://english.stackexchange.com/a/133950).

### CODE REVIEW

File: documents/policies/internal/Deceased Registered Speedcubers.md [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
-
-1. We should not attempt to administrate which speedcubers are deceased in WCA profiles. All the In Memoriam links shall be removed, and none will be added in the future, as speedcubers’ profiles are a separate matter.
-2. Public announcements about deceased speedcubers should be done as per below:
-   1. Public announcements are reserved only for speedcubers who contributed greatly to our organization. At the discretion of the WCA Board, we would consider people who performed activities for the WCA as Staff.
-   2. Public announcements would be displayed in the form of a post on our website and on our social media.
+1. WCA profiles should not contain any information related to the vital status of Registered Speedcubers.
+2. Public announcements about deceased speedcubers should be subject to the following conditions:
+   1. Be reserved only for speedcubers who contributed greatly to the WCA, at the discretion of the WCA Board.
+   2. Be displayed in the form of a post on the WCA website and WCA social media.
```
Comment (dmint789) [inline documents/policies/internal/Deceased Registered Speedcubers.md:12]: I'd just remove the second "WCA" on this line

Comment (Nanush7) [reply documents/policies/internal/Deceased Registered Speedcubers.md:12]: Line 12?

File: documents/policies/internal/Deceased Registered Speedcubers.md [hunk 2]
```diff
@@ -1,14 +1,12 @@
...
-
-1. We should not attempt to administrate which speedcubers are deceased in WCA profiles. All the In Memoriam links shall be removed, and none will be added in the future, as speedcubers’ profiles are a separate matter.
-2. Public announcements about deceased speedcubers should be done as per below:
-   1. Public announcements are reserved only for speedcubers who contributed greatly to our organization. At the discretion of the WCA Board, we would consider people who performed activities for the WCA as Staff.
-   2. Public announcements would be displayed in the form of a post on our website and on our social media.
+1. WCA profiles should not contain any information related to the vital status of Registered Speedcubers.
+2. Public announcements about deceased speedcubers should be subject to the following conditions:
+   1. Be reserved only for speedcubers who contributed greatly to the WCA, at the discretion of the WCA Board.
+   2. Be displayed in the form of a post on the WCA website and social media.
```
Comment (nsilvestri) [inline documents/policies/internal/Deceased Registered Speedcubers.md:12]: One small change for flexibility:

```suggestion
   2. Be displayed in the form of a post on the WCA website and/or social media.
```

**Review by nsilvestri — CHANGES_REQUESTED** [review]

**Review by dmint789 — APPROVED** [review]
LGTM!

File: documents/policies/internal/Deceased Registered Speedcubers.md (hunk 1 above)
Comment (dmint789) [reply documents/policies/internal/Deceased Registered Speedcubers.md]: Yeah

**Review by nsilvestri — APPROVED** [review]
Approved by Board vote

**Pull request merged**